### ISSUE TITLE
Tokenizer Quickstart Tutorial: Broken Links

### ISSUE THREAD

**Issue by SinaMostafanejad** [huggingface/tokenizers#1625]
## Summary

The **wikitext-103** links are broken both in the text and the first code block of the tokenizers' [Quicktour](https://github.com/huggingface/tokenizers/blob/main/docs/source-doc-builder/quicktour.mdx) tutorial.

**Comment by jbbqqf** [comment]
Hi! Triaging older issues — this one is partially fixed and the remaining piece is covered by an open PR, so it might be a candidate to close once that PR lands.

Status on current `main`:

- PR #1846 (merged 2025-08-29, commit [`ec54228`](https://github.com/huggingface/tokenizers/commit/ec54228)) already fixed the Salesforce blog link and the broken intra-doc `[this page](pipeline)` link in [`docs/source-doc-builder/quicktour.mdx`](https://github.com/huggingface/tokenizers/blob/main/docs/source-doc-builder/quicktour.mdx).
- The remaining dead link is the wget URL `https://s3.amazonaws.com/research.metamind.io/wikitext/wikitext-103-raw-v1.zip` (now 301 → 403). That's tracked separately as #1683.
- Open PR #2064 ("Replace dead wikitext s3 link in quicktour with HF dataset mirror", by @adityasingh2400, 2026-05-21) swaps the URL for the canonical `huggingface.co/datasets/mattdangerw/wikitext-103-raw/resolve/main/wikitext-103-raw-v1.zip` mirror and is explicitly tagged `Fixes #1625 Fixes #1683`.

Once #2064 merges, both this issue and #1683 will be auto-closed. Until then, leaving as-is is fine — there's no separate work needed here. @ArthurZucker, would you mind closing this out (either now as duplicate of #1683 + #2064, or after #2064 merges)?

---
*Disclosure: I drafted this comment with help from Claude Code while triaging stale issues; the PR / commit / link references above were verified manually.*
